### PR TITLE
W3 - D2 - LuanTran - Exercise Ruby Syntax Data Structure

### PR DESCRIPTION
**Pull request by mvn-luantran-dn**
Exercise for #13: 
Anh cho e nộp bài tập Ruby Syntax, Data structure. 
- [x] Check conventions
- [x] Exercise 1
- [x] Exercise 2
- [x] Exercise 3
Người review kiểm tra những danh sách sau trước khi merge
- [ ] Đúng conventions chưa?

### CODE REVIEW

File: at-luantran/Week3/exercise3.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Ruby data structure to hold this data.
+arr = [
+  {:name => "Paul Barry", :interest => ["Programming", "Networking", "Security", "Open Source", "Frameworks"]},
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L3"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L3"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L3"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L3"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  {:name => "Chris Meudec", :interest => ["Testing", "Safety Systems", "Formal Systems", "Programming Languages"]},
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  {:name => "Nigel Whyte", :interest => ["Graphics", "Imaging", "Programming", "Sign Language", "Trees"]},
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 2 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 3 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 2 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 3 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 2 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L4"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 3 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  {:name => "Austin Kinsella", :interest => ["Networks", "WANs", "Programming", "Macintosh", "Digital Photography"]},
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 3 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 4 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 3 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 4 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 3 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L5"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 4 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  {:name => "Gerry Moloney", :interest => ["Placement", "Employment", "Emerging Systems", "Web Development"]}
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 4 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 5 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 4 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L6"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 5 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Layout/SpaceInsideHashLiteralBraces: Space inside { missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Layout/SpaceInsideHashLiteralBraces: Space inside } missing. (https://github.com/rubocop-hq/ruby-style-guide#spaces-operators)</td>
    </tr>
  </tbody>
</table>

Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/rubocop-hq/ruby-style-guide#hash-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+]
+
+# Show list areas of interest of the first and third lecturers.
+puts "List areas of interest of the first lecturers:"
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L11"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb (hunk 5 above)
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/131d3fc3dabef10f6c0e874500b4009067cfc93c/at-luantran/Week3/exercise3.rb#L7"></span>Style/StringLiterals: Prefer single-quoted strings when you don’t need string interpolation or special symbols. (https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals)</td>
    </tr>
  </tbody>
</table>

File: at-luantran/Week3/exercise3.rb [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+list = []
+
+arr.each do |x|
+  sum += 1 if x[:interest].include?('Programming')
+  x[:interest].map! { |a| a == 'Programming' ? 'Web Developer' : a }
+  list += x[:interest]
+end
+
+# List interest
+list.uniq
```
Comment (at-dev-support-01) [inline at-luantran/Week3/exercise3.rb]: <table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/b1774a336172ac7a9a66aad474d2e865486a0eee/at-luantran/Week3/exercise3.rb#L35"></span>Layout/TrailingBlankLines: Final newline missing. (https://github.com/rubocop-hq/ruby-style-guide#newline-eof)</td>
    </tr>
  </tbody>
</table>

**Review by ghost — APPROVED** [review]

**Pull request merged**